App/HejDTO/Sources/HejDTO/Contests/GET/ContestDetail/Contests.Details.swift

```diff
@@ -0,0 +1,68 @@
+
+import Foundation
+
+extension Contests {
+    
+    ///https://docs.hejto.pl/#tag/Contests/operation/get_contest
+    public struct Detail: Codable, Equatable {
+        public let type, title, slug, content: String
+        public let startedAt, finishedAt: String
+        public let image: Common.Image
+        public let awards: [Award]
+        public let organizedBy: Contests.Organizer
+        public let userScore, userPosition: Int
+        public let links: Common.SelfLink
+        
+        public struct Award: Codable, Equatable {
+            public let title, slug, content: String
+            public let position: Int
+            public let images: [AwardImage]
+            
+            public struct AwardImage: Codable, Equatable {
```

Comment:
Just the `Image` is fine here. We know it's an `award image` because it's nested inside the `Award` type :) If you are confused about that please reach out to me :)